examples/knowledge_tuning/data-generation-with-llama-70b/synth_knowledge1.5_llama-7b.yaml

```
@@ -0,0 +1,136 @@
+- block_type: DuplicateColumns
+  block_config:
+    block_name: duplicate_document_col
+    columns_map:
+      document: base_document
+
+- block_type: LLMBlock
+  block_config:
+    block_name: gen_detailed_summary
+    config_path: configs/knowledge/detailed_summary.yaml
+    model_id: model/RedHatAI/Meta-Llama-3.1-8B-Instruct-quantized.w4a16 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)